build.sbt

```
@@ -82,6 +83,9 @@ def akkaDiscoveryDependency(name: String, excludeThese: ExclusionRule*) =
 def akkaPersistenceCassandraDependency(name: String, excludeThese: ExclusionRule*) =
   "com.typesafe.akka" %% name % AkkaPersistenceCassandraVersion excludeAll ((excludeTheseDependencies ++ excludeThese): _*)
 
+def akkaSBRDependency(name: String, excludeThese: ExclusionRule*) =
```

```suggestion
def akkaEnhancementsDependency(name: String, excludeThese: ExclusionRule*) =
```
How about `akkaEnhancementsDependency` if the name of the module is a parameter?

---

Thanks for the suggestion!